src/bootstrap.js

```diff
@@ -187,7 +200,7 @@ export class Bootstrap {
     // Uninstall in reverse order
     (plugins || this.__installed)
       .reverse()
-      .forEach((p) => this.__uninstall(p.__name));
+      .forEach((p) => this.__uninstall(p));
```

Comment:
I think this may have been an error introduced in #12... Seems like the intent was `uninstall` takes `string | plugin | string[] | plugin[]`...in the case of strings, `p.name` is not defined. 

(This was causing a failed test that called `threestrap.uninstall("cursor")`.

Reply:
Good catch, going over #12 now to see where I did this